src/libmongoc/doc/conf.py

```diff
@@ -40,6 +44,10 @@
 # Fedora package builds must work offline - maintain a recent copy here
 intersphinx_mapping = {
     "bson": ("https://www.mongoc.org/libbson/current", "libbson-objects.inv"),
+    # NOTE: Pinned to CMake version hear, but is safe to update as new CMake is released.
+    # This pin version ensures that documentation labels remain stable between builds.
+    # When updating, fix any moved/renamed references as applicable.
+    "cmake": ("https://cmake.org/cmake/help/v3.27", None),
```

Comment:
```suggestion
    "cmake": ("https://cmake.org/cmake/help/latest/", None),
```
Given the mongoc docs track latest, wouldn't it be more consistent (and convenient) for the CMake docs to also track latest?

Reply:
I would like to use `/latest/`, but my concern is about reference stability. The intersphinx extension downloads an `objects.inv` from the given base URL when Sphinx executes. If we point to a specific version, it is guaranteed that the returned inventory will be stable, while `/latest/` might spuriously break references we have in the docs. I'd like to keep it up-to-date regularly, though, hence the comment.